en/docs/learn/deploying-the-sample-app.md

```diff
@@ -519,6 +503,28 @@ The next step is to configure the service provider.
 
 ----------------
 
+### Deploy the sample web app
+
+Next, deploy the sample web app on a web container.
+
+1. Extract the `pickup-dispatch.war` file and open the `dispatch.properties` file located in the `<EXTRACT>/WEB-INF/classes` folder.
+
+2. Replace the `consumerKey` and `consumerSecret` values with the OAuth client key and OAuth client secret values that were generated for the newly created service provider.
+
+    ![pickup-key-secret](../assets/img/learn/pickup-key-secret.png)
+
+3. Next, copy the extracted and modified `pickup-dispatch` folder to the `<TOMCAT_HOME>/webapps` folder.
+
+4. Start the Tomcat server.
+
+To check the sample application, navigate to
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/pickup-dispatch/home.jsp         `
+on your browser.
+
+For example,
+`                     http://localhost:8080/pickup-dispatch/home.jsp                 .         `
```

Comment:
Is this localhost or localhost.com?

Reply:
Just 'localhost' works for the oidc pickup samples